woocommerce/woocommerce-gutenberg-products-block
# Add ShippingCalculator component

Fixes #1530.

Add a form to change the address used to calculate the estimated shipping costs in the _Cart_ block.

Notice there are follow-ups for updating the shipping cost dynamically (#1531) and for updating the `location` string (#1606). This PR is only about adding the form and hooking it up to the shipping rates displayed below.

### Screenshots
![imatge](https://user-images.githubusercontent.com/3616980/74543000-e9f8f980-4f44-11ea-88e4-611bf4dbd146.png)

### How to test the changes in this Pull Request:

1. Create a post with a _Cart_ block.
2. Click on the _Change address_ button.
3. Change the address and click on _Update_.
4. Verify shipping rates are updated according to your options.
